### PR TITLE
Fix for installation using zsh

### PR DESCRIPTION
**Pull request by ehsteve**
## PR Description

Small fix to installation instructions. Does not work on zsh which is the new terminal for Mac now.

### CODE REVIEW

File: docs/dev_guide/contents/newcomers.rst [hunk 1]
```diff
@@ -162,7 +162,7 @@ This will make submitting changes easier in the long term for you:
 
     $ git clone https://github.com/<username>/sunpy.git sunpy-git
     $ cd sunpy-git
-    $ pip install -e .[dev]
+    $ pip install -e ."[dev]"
```
Comment (nabobalis) [inline docs/dev_guide/contents/newcomers.rst:165]:
```suggestion
    $ pip install -e ".[dev]"
```

Can you check this works?

**Review by nabobalis — APPROVED** [review]

File: docs/dev_guide/contents/newcomers.rst (hunk 1 above)
Comment (ehsteve) [reply docs/dev_guide/contents/newcomers.rst]: That does work and is more consistent with the other docs.

**Comment by nabobalis** [comment]
Thanks @ehsteve

File: docs/dev_guide/contents/tests.rst [hunk 2]
```diff
@@ -23,7 +23,7 @@ Since the testing dependencies are not actually required to install or use sunpy
 
 Developers who want to run the test suite will need to install the testing packages using pip::
 
-    $ pip install -e .[tests]
+    $ pip install -e ".[tests]""
```
Comment (Cadair) [inline docs/dev_guide/contents/tests.rst:26]: Double quote is a typo?

Comment (nabobalis) [reply docs/dev_guide/contents/tests.rst:26]: How did I miss that?!

Comment (ehsteve) [reply docs/dev_guide/contents/tests.rst:26]: Oops!

Comment (ehsteve) [reply docs/dev_guide/contents/tests.rst:26]: Stupid double quote completion.

Comment (nabobalis) [reply docs/dev_guide/contents/tests.rst:26]: If you only accepted my suggestion

Comment (nabobalis) [reply docs/dev_guide/contents/tests.rst:26]: If you have time, you can throw up another PR otherwise I'll get to it when I end up at UIC.

**Pull request merged**